packages/cli/configuration/fern/definition/docs.yml

```diff
@@ -24,10 +24,88 @@ types:
   AnalyticsConfig:
     properties:
       segment: optional<SegmentConfig>
+      fullstory: optional<FullStoryAnalyticsConfig>
+      intercom: optional<IntercomConfig>
+      posthog: optional<PostHogConfig>
+      gtm: optional<GTMConfig>
+      ga4: optional<GoogleAnalytics4Config>
+      amplitude: optional<AmplitudeConfig>
+      mixpanel: optional<MixpanelConfig>
+      hotjar: optional<HotJarConfig>
+      koala: optional<KoalaConfig>
+      logrocket: optional<LogRocketConfig>
+      pirsch: optional<PirschConfig>
+      plausible: optional<PlausibleConfig>
+      fathom: optional<FathomConfig>
+      clearbit: optional<ClearBitConfig>
+      heap: optional<HeapConfig>
 
   SegmentConfig:
     properties:
       write-key: string
+  
+  FullStoryAnalyticsConfig:
+    properties:
+      orgId: string
+
+  IntercomConfig:
+    properties:
+      appId: string
+      apiBase: optional<string>
+
+  PostHogConfig:
+    properties:
+      apiKey: string
+      endpoint: optional<string>
+
+  GTMConfig:
+    properties:
+      containerId: string
+
+  GoogleAnalytics4Config:
+    properties:
+      measurementId: string
+
+  AmplitudeConfig:
+    properties:
+      apiKey: string
+
+  MixpanelConfig:
+    properties:
+      apiKey: string
+
+  HotJarConfig:
+    properties:
+      hjid: string
+      hjsv: string
+
+  KoalaConfig:
+    properties:
+      apiKey: string
+
+  LogRocketConfig:
+    properties:
+      apiKey: string
+
+  PirschConfig:
+    properties:
+      id: string
+
+  PlausibleConfig:
+    properties:
+      domain: string
+
+  FathomConfig:
+    properties:
+      siteId: string
+
+  ClearBitConfig:
+    properties:
+      apiKey: string
+
+  HeapConfig:
+    properties:
+      appId: string
```

Comment:
Everything else is kebab cased, b/c users will be writing config and all of our config is kebeb based